segmentstore/server/src/test/java/io/pravega/segmentstore/server/store/StreamSegmentStoreTestBase.java

```diff
@@ -187,8 +188,14 @@ public void testEndToEnd() throws Exception {
             log.info("Finished appending data.");
 
             checkSegmentStatus(lengths, startOffsets, false, false, expectedAttributeValue, segmentStore);
+            builder.close();
```

Comment:
When builder is java.lang.AutoCloseable and used in try block you don't need to call close explicitly. The compiler will automatically add that statement